### PR TITLE
Add docs on where to find the SLA reports

### PR DESCRIPTION
**Pull request by Kidswiss**
## Summary

A few lines on how to find the PDF files for the AppCat SLA reports.

## Checklist

- [x] Try to isolate changes into separate PRs (to build a better changelog).
- [x] Categorize the PR by setting a good title and adding one of the labels:
      `change`, `decision`, `requirement/quality`, `requirement/functional`, `dependency`
      as they show up in the changelog
- [x] Link this PR to related issues if applicable.

### CODE REVIEW

File: docs/modules/ROOT/pages/reference/slareports.adoc [hunk 1]
```diff
@@ -0,0 +1,8 @@
+= AppCat SLA Reports
+
+Currently the SLA reports for AppCat are generated on the first of each month and stored in an S3 bucket on objects.lpg.cloudscale.ch.
+The bucket name is `appcat-sla-reports`, the credentials are available the objectBucket in the namespace `syn-appcat-slos` on APPUiO Cloud.
```
Comment (lieneluksika) [inline docs/modules/ROOT/pages/reference/slareports.adoc:4]:
```suggestion
The bucket name is `appcat-sla-reports`. The credentials for the objectBucket are available in the namespace `syn-appcat-slos` on APPUiO Cloud.
```

File: docs/modules/ROOT/pages/reference/slareports.adoc [hunk 2]
```diff
@@ -0,0 +1,8 @@
+= AppCat SLA Reports
+
+Currently the SLA reports for AppCat are generated on the first of each month and stored in an S3 bucket on objects.lpg.cloudscale.ch.
```
Comment (tobru) [inline docs/modules/ROOT/pages/reference/slareports.adoc:3]:
```suggestion
Currently the SLA reports for AppCat are generated on the first day of each month and stored in an S3 bucket on objects.lpg.cloudscale.ch.
```

**Review by tobru — COMMENTED** [review]
Are we OK with providing so much information about the bucket location in the open? If yes, is the configuration on the bucket good enough to not leak information to unauthenticated users?

**Comment by Kidswiss** [comment]
> Are we OK with providing so much information about the bucket location in the open? If yes, is the configuration on the bucket good enough to not leak information to unauthenticated users?

I can just point to the secret where everything is stored. But yes the bucket itself is private, you can't see anything without authentication.

**Comment by tobru** [comment]
> I can just point to the secret where everything is stored

I'd do that, with that, the chances are also lower of having outdated docs

File: docs/modules/ROOT/pages/reference/slareports.adoc [hunk 3]
```diff
@@ -0,0 +1,8 @@
+= AppCat SLA Reports
+
+Currently, the SLA reports for AppCat are generated on the first of each month and stored in an S3 bucket.
```
Comment (tobru) [inline docs/modules/ROOT/pages/reference/slareports.adoc]:
```suggestion
Currently, the SLA reports for AppCat are generated on the first day of each month and are stored in an S3 bucket.
```

**Pull request merged**